tsconfig.json

```diff
@@ -13,6 +13,7 @@
 		// "noImplicitReturns": true, /* Report error when not all code paths in function return a value. */
 		// "noFallthroughCasesInSwitch": true, /* Report errors for fallthrough cases in switch statement. */
 		// "noUnusedParameters": true,  /* Report errors on unused parameters. */
+		"moduleResolution": "bundler"
```

Comment:
This was required to import arktype since arktype is an ES module. This also means we can get rid of the dynamic import for ansi-regex now but I won't do it in this PR.